core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/internal/signer/SignerLoader.java

```diff
@@ -29,20 +28,16 @@
 /**
  * Utility class for instantiating signers only if they're available on the class path.
  */
-@SdkInternalApi
+@SdkProtectedApi
```

Comment:
The SignerLoader itself is not used cross-module; used only by AwsV4aHttpSigner. This should remain SdkInternalApi.

Reply:
Sure, I moved it to protected to mirror the [existing one](https://github.com/aws/aws-sdk-java-v2/blob/233668bc00ebabffadd6a485b7dc27cbbed2ed27/core/auth/src/main/java/software/amazon/awssdk/auth/signer/SignerLoader.java#L29C1-L29C1), but we can relax this once and if needed. I will move it back to internal.

Reply:
fyi, the existing one is SdkProtectedApi, because it is used by generated {Service}EndpointAuthSchemeInterceptor.